File: CSharp/Contests2025/ABC407/B.cs
class B
{
	static int[] Read() => Array.ConvertAll(Console.ReadLine().Split(), int.Parse);
	static (int, int) Read2() { var a = Read(); return (a[0], a[1]); }
	static long[] ReadL() => Array.ConvertAll(Console.ReadLine().Split(), long.Parse);
	static void Main() => Console.WriteLine(Solve());
	static object Solve()
	{
		var (x, y) = Read2();

		var c = 0;
		for (int i = 1; i <= 6; i++)
			for (int j = 1; j <= 6; j++)
				if (i + j >= x || Math.Abs(i - j) >= y) c++;
		return c / 36.0;
	}
}

File: Java/Contests2025/ABC407/src/B.java
import java.util.Scanner;

public class B {
	static Scanner sc = new Scanner(System.in);

	public static void main(String[] args) {
		System.out.println(solve());
	}

	static Object solve() {
		var x = sc.nextInt();
		var y = sc.nextInt();

		var c = 0;
		for (int i = 1; i <= 6; i++)
			for (int j = 1; j <= 6; j++)
				if (i + j >= x || Math.abs(i - j) >= y)
					c++;
		return c / 36.0;
	}
}
